hub/core/tensor.py

```diff
@@ -95,15 +102,21 @@ def add_samples_to_tensor(
     for i in range(array_length):
         sample = array[i]
 
-        # TODO: we may want to call `tobytes` on `array` and call memoryview on that. this may depend on the access patterns we
-        # choose to optimize for.
-        b = memoryview(tobytes(sample))
+        if is_shape_empty(sample.shape):
+            # if sample has a 0 in the shape, no data will be written
+            index_map_entry = {"chunk_names": []}  # type: ignore
 
-        index_map_entry = write_bytes(
-            b, key, tensor_meta["chunk_size"], storage, index_map
-        )
+        else:
+            # TODO: we may want to call `tobytes` on `array` and call memoryview on that. this may depend on the access patterns we
+            # choose to optimize for.
+            b = memoryview(tobytes(sample))
+
+            index_map_entry = write_bytes(
```

Comment:
this return statement will be removed by @thisiseshan 's index map work

Reply:
Was it merged? I don't see any return statements...

Reply:
no i meant the return value